.github/workflows/conan-package.yml

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
WHy do we even need a package? That's outside the scope of getting the tests to work right? THere is a seperate run for the tests and that doesn't require this. We don't need cura engine to be packaged to run the tests. Cura isnt' depending on curaEngine to run those unit tests.

Reply:
It was added because this PR was to be part of a multiple PR's such as Cura, which consumes CuraEngine. Same way that Arcus is consumed by CuraEngine and need to run the tests of CuraEngine.
Not sure if CuraEngine is needed for the UnitTest of Cura. But this workflow stays roughly the same over all our repo's

I even thought that it could be converted to a GitHub action to be reused, but I first wanted to make it work

Reply:
> Not sure if CuraEngine is needed for the UnitTest of Cura.

I don't think so. (Also that would literally be an integration test then.)

Reply:
(Though Uranium _is_ needed.)